crates/client/src/commander.rs

```diff
@@ -405,7 +405,7 @@ impl Commander {
         let program_idls_codes = self.program_packages().map(|package| async move {
             let name = package.name;
             let output = Command::new("cargo")
-                .arg("+nightly")
+                .arg("+nightly-2023-12-28-x86_64-unknown-linux-gnu")
```

Comment:
Why do we need a specific nightly version? It works for me If I use only +nightly